README.md

```diff
@@ -297,29 +282,17 @@ MoonSDK.SendLevelDataStartEvent(levelIndex, coinsAmount, purchaseIDs);
 #### **`LevelDataCompleteEvent`**  
 This event is sent at the end of a level and contains the following information:
 
-1. **`LevelStatus`** – The current status of the level:
-   - `"start"` – When the level begins.
-   - `"fail"` – If the player fails the level.
-   - `"complete"` – If the player completes the level.
-   
-2. **`levelIndex`** – The index of the level. Send this as an integer (e.g., `1`), which the SDK will convert to Game Analytics format (e.g., `0001`). Make sure the levels start from `1` (not `0`).
-
-3. **`LevelResult`** – The result of the level, either `"win"` or `"fail"`.
-
-4. **`isContinue`** – A boolean indicating whether the player is continuing from where they left off (`true`) or starting fresh (`false`). For idle or revive mechanics:
-   - If a player enters the level for the first time, set `isContinue = false`.
-   - If a player restarts the level or resumes from saved progress, set `isContinue = true`.
-   - If your game doesn't have this feature, set this to `false` by default.
-
-5. **`coinsAmount`** – The player's balance of the main currency at the end of the level.  
-   - Example: If the player started the level with 100 coins and earned 50 during the level, send `150`. If the player started with 900 coins and earned 15, send `915`. Make sure to send the event *after* updating the player's new balance.
-
-7. **`movesAmount`** – The total number of moves the player made during the level.
+1. **`levelNumber`** - Indicates the current level number
+2. **`result`** - Represents the outcome of the level, which could be "win" if the player successfully completes it or "fail" if the player fails to complete it.
+3. **`isContinueLevel`** - A boolean argument that indicates whether the player is continuing the level from where they left off (true) or starting it from the beginning (false). This is particularly useful for long idle levels or when there's a revive option. If the game doesn't have these features, it should be set to false by default.
+4. **`coinsAmount`** - Represents the amount of soft currency when the level is finished
```

Comment:
We need the explanations. "- Example: If the player started the level with 100 coins and earned 50 during the level, send `150`. If the player started with 900 coins and earned 15, send `915`. Make sure to send the event *after* updating the player's new balance."